go/kbfs/libdokan/fs.go

```diff
@@ -367,31 +367,16 @@ func (f *FS) open(ctx context.Context, oc *openContext, ps []string) (dokan.File
 		oc.isUppercasePath = true
 		fallthrough
 	case PublicName == ps[0]:
-		// Refuse private directories while we are in a a generic error state.
-		if f.remoteStatus.ExtraFileName() == libfs.HumanErrorFileName {
-			f.log.CWarningf(ctx, "Refusing access to public directory while errors are present!")
-			return nil, 0, dokan.ErrAccessDenied
-		}
 		return f.root.public.open(ctx, oc, ps[1:])
 	case strings.ToUpper(PrivateName) == ps[0]:
 		oc.isUppercasePath = true
 		fallthrough
 	case PrivateName == ps[0]:
-		// Refuse private directories while we are in a error state.
-		if f.remoteStatus.ExtraFileName() != "" {
```

Comment:
Does ` != ""` do the same thing as ` == libfs.HumanErrorFileName` would do here? If so, the latter seems more readable. If not, maybe can you make a helper function?

Reply:
I simplified the logic even more.